pandas/indexes/multi.py

```diff
@@ -864,9 +862,9 @@ def from_arrays(cls, arrays, sortorder=None, names=None):
             if len(arrays[i]) != len(arrays[i - 1]):
                 raise ValueError('all arrays must be same length')
 
-        cats = [Categorical.from_array(arr, ordered=True) for arr in arrays]
```

Comment:
OK. I'll try to do it today if it goes smoothly. Otherwise, I'll do a follow up as soon as this PR is merged in.

 Just a question:
`Categorical.from_array` should emit a `FutureWarning` with a comment like this: "Categorical.from_array is deprecated, use Categorical instead"?